Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A4
  # add  github.com/julien-noblet/download-geofabrik
 RUN go get github.com/julien-noblet/download-geofabrik \
  && go install  github.com/julien-noblet/download-geofabrik \
- && download-geofabrik update \
+ && download-geofabrik generate \
```

Comment:
As I see - not in the master -repo  yet ( https://github.com/julien-noblet/download-geofabrik/tree/master )

```
$ docker run -it golang bash
root@6d59127aff4f:/go# go get github.com/julien-noblet/download-geofabrik 
root@6d59127aff4f:/go# go install  github.com/julien-noblet/download-geofabrik 
root@6d59127aff4f:/go# download-geofabrik generate
download-geofabrik: error: expected command but got "generate", try --help
```

Reply:
Oh!!! :cry: 
Yes you are true, I haven't pushed v2.2.0 yet... 
I've made this PR too early ^^
I will push it today.